test/AndcultureCode.CSharp.Sitefinity.Core.Tests/BaseUnitTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Xunit;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Tests
+{
+    [Trait("Category", "Unit")]
+    public class BaseUnitTest
```

Comment:
Just wondering if we're planning on using this.  I don't see it in any of our project unit tests (because we don't have any yet).  Wondering what the `Trait("Category", "Unit")` does for us when we inherit from this base?

Reply:
We have this in our other OSS repos, and the value is that you can categorize tests by trait in VS and I believe run them by trait, vs. having to filter based on some wildcard string matcher.

Reply:
Not sure if it was intentional, but we do have some barebones testing classes available from the `AndcultureCode.CSharp.Testing` package. You could pull that in, and make your own base if anything custom is needed.

Reply:
👍

Reply:
> Not sure if it was intentional, but we do have some barebones testing classes available from the `AndcultureCode.CSharp.Testing` package. You could pull that in, and make your own base if anything custom is needed.

My only thought was there's a lot of additional stuff in that package that we don't need (ex: all the AspNetCore references), and lots of the stuff in that repo likely won't apply to Sitefinity -- e.g. the IntegrationTestBase likely won't be useful as up to this point we don't have repositories in our SF project, because use the managers they provide to grab data. That being said, it may end up being useful as we sink our teeth more into Sitefinity development, but for now it feels like overkill.